sss/sss_table.py

```diff
@@ -272,46 +450,32 @@ def data_folder_to_database(data_folder, db_url=default_db_url):
         # read file and conduct pre-processing
         df, file = read_file(i)
 
-        #clarify from cheng what will be defined as miscellaneous, and health_care
-
-        df, miscellaneous, health_care, arpa = check_extra_columns(df)
-        # need to have more precise solution for this specific problem
-        # df['infant'] = pd.to_numeric(df['infant'],errors='coerce')
-        # df['emergency_savings'] = pd.to_numeric(
-        #                           df['emergency_savings'],errors='coerce')
-
-        # Create a 'weighted_child_count' column from a*c* values
-        #   this will take the infant count and move to this column
-        df['weighted_child_count'] = df['infant']
-        df.loc[df.loc[:, 'family_type'].isin(
-            [i for i in df['family_type']
-                if 'c' not in i]),
-                'weighted_child_count'] = np.nan
+        # store dataframes created
+        df, arpa, health_care,  miscellaneous  = check_extra_columns(df)
 
-        # removing duplicate rows, want to move this
-        df = df.drop_duplicates(subset=['analysis_type', 'family_type',
-                                        'state', 'year', 'place'])
-        df.reset_index(inplace=True, drop=True)
+        # update the primary table df
+        df = prepare_for_database(df)
 
+        # prints what file we are reading
         print(file.split('/')[-1])
```

Comment:
It might help users to make the printout a little more clear. Maybe we could add something "processing file" before the filename. Also, I'd consider moving the printout to below the `bulk_insert` call so that it only gets printed after the data are actually in the database. If you do that, the text before the filename could be "inserted data from" or something similar.